src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers High Availability Node-RED
+subtitle: FlowForge 1.8 makes Node-RED applications more reliable and scalable, plus more streamline deployment pipelines.
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+image: /blog/2023/06/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## More reliable and scalable Node-RED applications
+Do you want to run business-critical processes in Node-RED and need to ensure they are always available and can handle increasing workloads? We're excited to introduce High Availability (HA) in FlowForge to support your needs. Suppose you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are evenly distributed across two instances. This means your applications are less likely to experience downtime and can manage higher traffic loads. You can easily activate HA mode during the creation of an instance. For more details, please see our [documentation](https://flowforge.com/docs/user/).
+
+High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
```

Comment:
We should be linking to documentation on how to try it out?

Reply:
Absolutely, it's just a placeholder for the tests. The documentation is not ready yet.

Reply:
Please also offer them a 30 day trial license for their self-managed installs on Kubernetes by booking a demo at flowforge.com/book-demo

Reply:
See: https://github.com/flowforge/website/pull/820#discussion_r1222579519